lib/mayaUsd/resources/ae/usdschemabase/attribute_custom_control.py

```diff
@@ -32,6 +32,14 @@ def getNiceAttributeName(ufeAttr, attrName):
         attrName = str(ufeAttr.getMetadata("uiname"))
     return mayaUsd.lib.Util.prettifyName(attrName)
 
+def cleanAndFormatTooltip(s):
+    # Remove leading/trailing whitespace and replace newlines.
+    lines = s.splitlines()
+    stripped = [line.strip() for line in lines]
+    cleaned = '<br>'.join(stripped)
+
+    # Don't allow the tooltip to word-wrap.
+    return "<p style='white-space:pre'>" + cleaned + '</p>'
```

Comment:
splitlines() returns list splitting string at line breaks. Then create new list by removing spaces at beginning and end of strings. Then create one string using <br> instead of newline. Finally don't allow tooltip to wrap, as it contains breaks.

I think this is better than what UsdView is doing since it also removes the single leading space that subsequent lines have.